readme.md

```diff
@@ -2,29 +2,92 @@
+
+#### Lower cost and complexity
+Letterbook doesn't maintain any live state. Everything lives in the database, so there's no need to run a Redis cluster or similar to act as shared state storage. We also expect to have significantly lower compute demands, due to both the architectural choices to avoid expensive infrastructure and system sprawl, and the use of C#, a very high performance compiled language.
+
+#### First class observability
+Letterbook is thoroughly instrumented for both automatic and custom telemetry, including robust logging, metrics, and distributed tracing. We also provide out-of-the-box collection and dashboards for our telemetry. You can investigate errors, bugs, and performance issues the same way we the developers would.
+
+### [For Moderators](https://github.com/Letterbook/Letterbook/issues/132)
+#### Local-only posts
+Avoid context collapse! You can talk to your users and they can talk to each other, without risking context collapse by exposure to the whole fediverse.
+
+#### Automatically expiring actions
+Moderator actions like blocking, muting, and limiting federation will all be able to automatically expire after a set time.
+
+#### Fine grained federation controls
+You'll be able to do things like prevent federated posts from appearing in promoted feeds, prevent them from appearing at all without an established follow relationship, hide posts behind a click-through and warning, and defederate without breaking your users follow relationships, in addition to the same basic options as other servers.
+
+#### Other moderator tools
+- Keep and share notes
+- Audit logs
+- Auditable privileged views of non-public posts
+- Spam and quality filters
+- And more
 
-I hate the term microblogging, but I love the dynamic of it. Or at least the dynamic it can facilitate. I prefer to call it open correspondence. In fact, that open correspondence is the origin of the name. In the times when written letters were the standard technology for communicating across distances, a letter book was an actual book used to store and file those letters. Letterbook is where you keep your correspondence. Out of all the options which support that kind of open correspondence, I believe the fediverse has the most upside, and by far the most future potential. But I have concerns about the sustainability of the fediverse as it stands now, and I'm hardly the only one. Topics like cost, admin burnout, and haphazard moderation come up frequently. Letterbook is an attempt to address those concerns.
+### [For Members](https://github.com/Letterbook/Letterbook/issues/133)
+#### Frequently requested features
+- Quote replies
+- Collapse notifications
+- Block, limit, and remove replies to your posts
+- Propose and accept edits to alt text and content warnings
+- Compose multi-post threads
+- Save drafts and scheduled posts
+- Formatted posts (bold, italics, lists, and block quotes, at least)
```

Comment:
maybe specify the supported formatting (markdown or whatever) instead of specific styles? idk, it probably doesn't matter.